addons/portal/static/src/chatter/frontend/portal_chatter.js

```diff
@@ -18,5 +19,26 @@ export class PortalChatter extends Component {
             inFrontendPortalChatter: true,
         });
         this.overlayService = useService("overlay");
+        this.store = useService("mail.store");
+        this.env.bus.addEventListener("reload_chatter_content", ev => this._reloadChatterContent(ev.detail));
+    }
+
+    async _reloadChatterContent(data) {
+        const thread = this.store.Thread.insert({
+            model: this.props.resModel,
+            id: this.props.resId,
+        });
+        const result = await rpc("/mail/chatter_fetch", {
+            ...thread.getFetchParams(),
+            limit: this.store.FETCH_LIMIT,
+        });
+        thread.messages = this.preprocessMessages(result.data["mail.message"]);
+    }
+
+    preprocessMessages(messages) {
+        return messages.map((m) => ({
+            ...m,
+            body: markup(m.body),
+        }));
```

Comment:
I think you can shorten the code with that :

```suggestion
    async _reloadChatterContent(data) {
        const thread = this.store.Thread.getOrFetch({
            model: this.props.resModel,
            id: this.props.resId,
        });
        thread.messages = await thread.fetchMessages();
    }
```

Reply:
`getOrFetch` is an asynchronous method (need to be awaited), but I think the thread is already being inserted here via the chatter service so no need to `getOrFetch` or insert? Just `get`?